.github/workflows/e2e.yaml

```diff
@@ -27,10 +34,11 @@ jobs:
         with:
           go-version: "1.22.11"
           cache: true
-      - run: yarn --frozen-lockfile
+      - name: Install node-canvas
+        run: sudo apt-get update && sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
This is for cypress, not our frontend. This is not required for amd64, but I added there as well just in case